### PR TITLE
Add in batch shuffling to `ConstantLengthDataset`

### PR DESCRIPTION
**Pull request by lvwerra**
This should make training a bit smoother.

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -151,6 +155,7 @@ def create_datasets(tokenizer, args):
         dataset = dataset.train_test_split(test_size=0.005, seed=args.seed)
         train_data = dataset["train"]
         valid_data = dataset["test"]
+        train_data = train_data.shuffle(seed=args.seed)
```
Comment (loubnabnl) [inline train.py:158]: we probably don't need this as the data is already shuffled with `train_test_split`?

**Review by loubnabnl — APPROVED** [review]
Thanks you ! 🤗
